0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+
+# Mapping
+
+We use QGIS and Mergin Maps for our fish passage data management and analysis. This section will get you started with our mapping procedures.
+
+## QGIS
```

Comment:
This can go after the mergin and phone setup.  come crew members don't need to download or learn qgis